va/va.h

```diff
@@ -4208,11 +4210,17 @@ typedef enum {
 } VADecodeErrorType;
 
 /**
- * Client calls vaQuerySurfaceError with VA_STATUS_ERROR_DECODING_ERROR, server side returns
+ * Client calls vaQuerySurfaceError() with the VAStatus obtained from vaSyncSurface(), server side returns
  * an array of structure VASurfaceDecodeMBErrors, and the array is terminated by setting status=-1
 */
 typedef struct _VASurfaceDecodeMBErrors {
-    int32_t status; /* 1 if hardware has returned detailed info below, -1 means this record is invalid */
+    /**
+     * Status reported by server side:
```

Comment:
server?

Reply:
it is what called in libva comments. Better to change to applicate side and driver side

Reply:
> it is what called in libva comments. Better to change to applicate side and driver side

Yes, I saw that `server` is used in several places in `va.h`, Do I need to change all the description or just this one?

Reply:
just this one, thanks ,  " status reported by backend driver"